### PR TITLE
add initial file for stale bot

### PR DESCRIPTION
**Pull request by codebytere**
let's talk about what we'd like to implement around marking and closing stale issues!

### CODE REVIEW

**Comment by malept** [comment]
Whatever we end up doing, we should add a section to `CONTRIBUTING.md` for a human-readable version of the policy.

File: .github/stale.yml [hunk 1]
```diff
@@ -0,0 +1,18 @@
+# Number of days of inactivity before an issue becomes stale
+daysUntilStale: 45
+# Number of days of inactivity before a stale issue is closed
+daysUntilClose: 7
+# Issues with these labels will never be considered stale
+exemptLabels:
+  - pinned
+  - security
```
Comment (zeke) [inline .github/stale.yml]: Looking at https://github.com/electron/electron/labels, I see the following that should probably be exempt:

```
 - fixme/bug
 - fixme/crash
 - fixme/regression
 - fixme/security
 - blocked
 - blocking-stable
 - needs-review
 ```

File: .github/stale.yml [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+# Label to use when marking an issue as stale
+staleLabel: stale
+# Comment to post when marking an issue as stale. Set to `false` to disable
+markComment: >
+  This issue has been automatically marked as stale because it has not had
+  recent activity and is not currently on maintainers' radars. It will be closed
```
Comment (zeke) [inline .github/stale.yml]: "not currently on maintainers' radars" -> "not currently prioritized"

File: .github/stale.yml [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+  in a week if no further activity occurs :)
+
+# Comment to post when closing a stale issue. Set to `false` to disable
+closeComment: false
```
Comment (zeke) [inline .github/stale.yml]: When an issue is closed I think we should say something like "Please leave a comment if this issue is still relevant."

**Comment by zeke** [comment]
@codebytere you were asking about our past use of the `stale` probot plugin: In the early days of probot (not so long ago), we installed the `stale` plugin on the @electron org and it went berserk, marking many issues across the org as stale, before we even had a chance to configure it. That took a while to clean up, and also revealed some race condition bugs in github.com itself, so it scared us and we turned it off. But the probot default behaviors have since improved so I'm 👍  on this change.

**Comment by codebytere** [comment]
I pushed up some config changes, and I think i saw some of the issues you're referencing in old issues that had `wontfix` splattered all over them haha

**Comment by codebytere** [comment]
On a related note, i'm going to try and implement https://github.com/behaviorbot/request-info/issues/8 in the service of our issue tab.

**Pull request merged**